### ISSUE TITLE
Support CraftTweaker's IngredientOr

### ISSUE THREAD

**Issue by ChiriVulpes** [codetaylor/artisan-worktables#68]
[CraftTweaker supports an "or" construct for providing multiple valid items without using the ore dictionary.](http://crafttweaker.readthedocs.io/en/latest/#Vanilla/Variable_Types/IIngredient#or-ing-an-iingredient)

Using this in an Artisan Worktables recipe works, but doesn't appear in JEI. I believe this can be fixed by modifying mtlib in Athenaeum. https://github.com/codetaylor/athenaeum/blob/master/src/main/java/com/codetaylor/mc/athenaeum/integration/crafttweaker/mtlib/helpers/CTInputHelper.java#L376-L397

Let me know if I should open up an issue over there.

**Comment by codetaylor** [comment]
Just one issue here is fine, thanks.

**Comment by codetaylor** [comment]
Verified working with:

```js
import mods.artisanworktables.Worktable;

Worktable.createRecipeBuilder("basic")
	.setShapeless([<minecraft:stick>.or(<minecraft:string>)])
	.setSecondaryIngredients([<minecraft:stick>.or(<ore:plankWood>)])
	.addTool(<ore:carpenters_handsaw>.or(<ore:engineers_driver>), 1)
	.addOutput(<minecraft:diamond>)
	.create();
```

**Comment by codetaylor** [comment]
Available in the alpha snapshot 1.12.2-1.15.25-10-gcce9f8f

Will be available in the beta 1.12.2-1.16.x
